keycloak/keycloak-admin-ui
# Realm settings: Sessions tab

## Motivation

https://issues.redhat.com/browse/APPDUX-938

## Brief Description

Adds sessions tab to realm settings section

## Verification Steps



1. Go to Realm settings -> sessions
2. Populate the fields with test values
3. Save and navigate to a different page
4. Verify the values are saved correctly

## Checklist:

- [x] Type checking has been performed via 'npm run check-types'

## Additional Notes
